contracts/TrueReward/utilities/IExponentContract.sol

```diff
@@ -1,9 +0,0 @@
-pragma solidity 0.5.13;
```

Comment:
We probably want to preserve the IExponentContract and have FractionalExponents extend from this.. We want to be able to swap out the existing exponent contract if we decide to use a new one.

Reply:
and make it an interface